nifi-nar-bundles/nifi-websocket-bundle/nifi-websocket-services-jetty/src/main/java/org/apache/nifi/websocket/jetty/JettyWebSocketClient.java

```diff
@@ -248,6 +268,44 @@ public void startClient(final ConfigurationContext context) throws Exception {
         }, sessionMaintenanceInterval, sessionMaintenanceInterval, TimeUnit.MILLISECONDS);
     }
 
+    private URI getEndpoint(final ConfigurationContext context) throws URISyntaxException {
+        final StringBuilder result = new StringBuilder(context.getProperty(WS_URI).evaluateAttributeExpressions(new HashMap<>()).getValue());
```

Comment:
The `evaluateAttributeExpressions()` method can be called without an argument, so it looks like the empty `HashMap` can be removed:
```suggestion
        final StringBuilder result = new StringBuilder(context.getProperty(WS_URI).evaluateAttributeExpressions().getValue());
```